### DISCUSSION TITLE
第5节课作业(7班)

### DISCUSSION THREAD

**Discussion by JimmyMa99** [InternLM/Tutorial#337] [作业]
第5节课作业(7班)

**Comment by wuhaostudio** [comment]
基础作业：
<img width="826" alt="微信图片_20240113194201" src="https://github.com/InternLM/tutorial/assets/143194075/3a81f254-cab3-474b-976d-40352020a0dc">
<img width="1070" alt="微信图片_20240114094256" src="https://github.com/InternLM/tutorial/assets/143194075/aab3f8cb-5bc6-4f8a-bdc6-3a83f82d7ffb">
<img width="1067" alt="微信图片_20240113191925" src="https://github.com/InternLM/tutorial/assets/143194075/dd2a6b82-0e18-4abd-9359-e1fc67532952">

**Comment by createup2023** [comment]
https://www.yuque.com/creatup/ynrsde/hf43xq879r8d6dg2?singleDoc# 《第五次作业：》
![image](https://github.com/InternLM/tutorial/assets/128460366/1f2fdd59-8e3d-4c60-966d-53d244361e16)

**Comment by lifelinezhang** [comment]
https://zhuanlan.zhihu.com/p/677829763

**Comment by SaaRaaS-1300** [comment]
第五节课作业同笔记，github链接：https://github.com/SaaRaaS-1300/InternLM_openNotebook/blob/main/Lesson-5/Lesson-5-Notebook.md

**Comment by anneeeee** [comment]
本地对话：
![image](https://github.com/InternLM/tutorial/assets/16276686/795c8fd1-09b5-4b13-a9e4-b0070ad82bea)

API：
![image](https://github.com/InternLM/tutorial/assets/16276686/69b9d28c-eabc-49b3-bc10-4f43cb0d0357)


Fast API的使用：
![image](https://github.com/InternLM/tutorial/assets/16276686/758e535b-1244-4250-a337-b87d7a6a0942)

Gradio页面：
![image](https://github.com/InternLM/tutorial/assets/16276686/7c2279ce-e500-4633-8128-e7adda73c6e9)

**Comment by hui1feng** [comment]
第五次课作业：
网页Gradio：
![5 5](https://github.com/InternLM/tutorial/assets/126125104/957bbae4-11c5-4de3-babe-23314832e748)

**Comment by PommesPeter** [comment]
本地对话：
![image](https://github.com/InternLM/tutorial/assets/54879512/c2c86148-3147-410c-b007-4494e3d9c727)

API 部署：
![image](https://github.com/InternLM/tutorial/assets/54879512/658b6264-8ca5-450d-b81f-bd1dabe0c737)

Gradio：
![image](https://github.com/InternLM/tutorial/assets/54879512/3d1805bd-a5b8-4f7a-a95a-72384eb14062)

**Comment by abelezm** [comment]
第五次课笔记同作业：https://blog.csdn.net/weixin_38043453/article/details/135612915

**Comment by leochiang** [comment]
第五次课作业
<img width="699" alt="11111" src="https://github.com/InternLM/tutorial/assets/7628117/3a406fc2-88bb-4eb7-b277-8673e880acb2">

**Comment by codeview126** [comment]
《LMDeploy部署》
2.2 TurboMind 推理+命令行本地对话： #稍后待截屏&补充
2.3 TurboMind推理+API服务：
2.4.1 TurboMind 服务作为后端： #暂未成功，等待再试
2.4.2 TurboMind 推理作为后端：
![20240123_07](https://github.com/InternLM/tutorial/assets/127104394/66e65299-1de9-48f1-8936-67e0544b3a47)

**Comment by Rogue-Trader-zzy** [comment]
作业：
![image](https://github.com/InternLM/tutorial/assets/64819379/a080813a-51d6-42ac-acaf-393643081454)
本地推理
后面再补更全一些。

**Comment by Simon-lyz** [comment]
## InternLM Lesson-5 量化部署
https://g4w90egc30.feishu.cn/docx/S7lPdG6F5ojQcxxFhQ1cYSC6nnh?from=from_copylink
